src/test/java/org/cactoos/scalar/AndInThreadsTest.java

```diff
@@ -46,6 +51,7 @@
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
+@SuppressWarnings("PMD.TooManyMethods")
 public final class AndInThreadsTest {
```

Comment:
@driver733 Please add javadocs over the tests, what is the input, what is the expected output and why? -- although I've seen that in general cactoos did not require test JavaDocs, I don't really understand how class ``AndInThreads`` is used on something else than Booleans. For instance, how does it work with a List of integers? It is not really clear from the test code alone.

Reply:
@amihaiemil Can I write failure explanation inside the tests instead?
I also did not get how this class works with various inputs from the first sight. I belive that the documentation of the AndInThreads class needs to be improved.

Reply:
@driver733 

>Can I write failure explanation inside the tests instead?

Yes, that could also work. 

>I belive that the documentation of the AndInThreads class needs to be improved.

Then go ahead and open an Issue about it.